functions/__fzf_search_git_log.fish

```diff
@@ -6,11 +6,11 @@ function __fzf_search_git_log --description "Search the output of git log and pr
         # See similar comment in __fzf_search_shell_variables.fish.
         set --local --export SHELL (command --search fish)
 
-        set log_fmt_str '%C(bold blue)%h%C(reset) - %C(cyan)%as%C(reset) %C(yellow)%d%C(reset) %C(normal)%s%C(reset)  %C(dim normal)[%an]%C(reset)'
+        # see documentation for git format placeholders at https://git-scm.com/docs/git-log#Documentation/git-log.txt-emnem
```

Comment:
I forgot to move this comment when I broke out the log format into its own line in 5ce8a51b78abf40468c96c06fd7c896bea7d05c0